portfolio/.gitignore

```diff
@@ -0,0 +1 @@
+config.js
```

Comment:
When adding an uncommitted config file to a repo, I like to leave a template or instructions in a README about what is expected to exist in that config file, so that other developers can replicate it. It would also be helpful for this .gitignore file to list the full path to the file we're ignoring, which I suspect is src/main/webapp/config.js, right?

From looking at the code, I suspect the file needs to contain something like:

```
var config = {MAPS_KEY: 'my-api-key-for-google-maps'};
```

Reply:
Sounds good! You're right on the file path, should I also include that on the README file or would it be better to include it in comments within the config.js file?